server.js

```diff
@@ -0,0 +1,59 @@
+const http = require('http');
+const fs = require('fs');
+//const public = require('./public')
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((request, response) => {
+  const uri = request.url;
+  const reqMeth = request.method;
+console.log(uri)
+  function reqResp(uri, type) {
+    fs.readFile(`./public${uri}.html`, (err, data) => {
+      if (err) throw 'error';
```

Comment:
Your readFile will throw an Error if the file does not exist, you will need to use this error to know when you send a 404.